tests/integration/telemetry/stackdriver/testdata/stackdriver_filter.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
We should be installing this from istioctl not another file, imo. This solves the duplication issue and tests what we are actually shipping to users.

How to do that is another question though. Right now we are running presubmit using istioctl not helm, but we have postsubmits running helm so it would work there. We may be able to run `manifest generate` and extract it from that but it seems pretty hacky... @richardwxn any ideas for extracting a single object from istioctl?

Reply:
I was thinking we could just run presubmit to use istioctl as well by adding that operator env variable that you added to telemetry prow job. We don't need to worry about helm installation. Helm charts do not have those envoy filters anyway.

Reply:
Agree we could just run this when istioctl is used, but not sure we have a good way to do that right now. We may need to add another Label for IstioctlOnly or something. Its not quite as straightforward as presubmit vs postsubmit, we run tests that are labelled as "presubmit" in postsubmit using helm.

Reply:
We could always run telemetry suite with istioctl by adding `TEST_USE_OPERATOR` (https://github.com/istio/istio/blob/master/tests/integration/tests.mk#L44) to the job right?

Reply:
@bianpengyuan in the order of development which config is authoritative?
When we write it the order was proxy (xds config in proxy) —> istio  integration tests (EnvoyFilter config) and then finally it graduated to the installer.

If we make a change to filter config we should be able to update the integration tests.  This flow seems backwards to me, we can’t have installer before integration.

Reply:
This is a general problem with the installer, which is why we agreed to move it into istio/istio so the integration tests are run at the same time

Reply:
We should treat what we ship to user as authoritative, and make sure it is tested. So for now we'd need to update installer/operator before updating proxy sha and integration test. It is painful but gives us more confidence on release.

Reply:
@howardjohn you mean extract a specific object from the generated manifests? there is no such a command now and might be hacky